### PR TITLE
CodeReady Containers option

### PR DESCRIPTION
**Pull request by AustinSeto**
Small update to intro section. Adds a blurb about using CRC, as part of https://github.com/OpenLiberty/guides-common/issues/467

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -35,27 +35,37 @@ Minishift.
...
 
 Using Maven, you will build the `system` microservice that collects basic system properties from your system and the `inventory` microservice that will 
 interact with the `system` microservice. Then, you will learn how to deploy both to the cluster and establish communication between them.
 
-You will use Minishift, a tool for you to run OKD on a local system. Minishift allows developers to deploy a quick and easy OKD cluster for application development.
+You will use Minishift, a tool for you to run OKD on a local system. 
+Minishift allows developers to deploy a quick and easy OKD cluster for application development.
+
+Minishift is based on OKD 3.11.
+To run OKD 4.1 or newer on your local system, you can instead use the CodeReady Containers tool.
```
Comment (dmuelle) [inline README.adoc]: To run OKD 4.1 or newer on your local system, you can instead use the CodeReady Containers tool.
--->
To run OKD 4.1 or newer on your local system, you can use the CodeReady Containers tool instead.

File: README.adoc [hunk 2]
```diff
@@ -241,17 +252,19 @@ In the `start` directory, run the following command to package the `system` micr
...
+and it is now ready to be injected into a Docker container for deployment.
 
 == Containerizing the system microservice
 
 === Reusing the Docker daemon
 
-To simplify the local deployment process, the built-in Minishift Docker daemon can be reused. Reusing the Minishift Docker daemon allows your system to 
-connect to it, and use the internal Docker registry. This avoids having to build a Docker registry on your machine. To reuse the Docker daemon, run the 
-following command to point your command-line session to use the Minishift's daemon:
+To simplify the local deployment process, the built-in Minishift Docker daemon can be reused.
```
Comment (dmuelle) [inline README.adoc]: suggested edit lines 264-66

To simplify the local deployment process, you can reuse the built-in Minishift Docker daemon.
Reusing the Minishift Docker daemon allows you to use the internal Docker registry, so you don't have to build a Docker registry on your machine.

File: README.adoc [hunk 3]
```diff
@@ -329,9 +342,9 @@ system                 latest              e8a8393e9364        2 minutes ago
 
 === Accessing the internal registry
 
-To run the microservice on the OKD cluster, you need to push the microservice image into a container image registry. You will use OpenShift’s 
-integrated container image registry called OpenShift Container Registry (OCR). First, you must authenticate your Docker client to your OCR. Start by running 
-the login command:
+To run the microservice on the OKD cluster, you need to push the microservice image into a container image registry.
+You will use OpenShift’s integrated container image registry called OpenShift Container Registry (OCR).
```
Comment (dmuelle) [inline README.adoc]: Avoid possessive for with products/inanimate objects. Suggested edit:

You will use OpenShift’s integrated container image registry
--->
You will use the OpenShift integrated container image registry

File: README.adoc [hunk 4]
```diff
@@ -598,8 +617,9 @@ deploy.yaml
 include::finish/deploy.yaml[]
 ----
 
-Now that the `system` microservice is running, you will package and deploy the `inventory` microservice, which adds the properties from the `system` 
-microservice to the `inventory`. This process demonstrates how communication can be established between pods inside a cluster.
+Now that the `system` microservice is running, you will package and deploy the `inventory` microservice,
+which adds the properties from the `system` microservice to the `inventory`.
+This process demonstrates how communication can be established between pods inside a cluster.
```
Comment (dmuelle) [inline README.adoc]: This process demonstrates how communication can be established between pods inside a cluster.
--->
This process demonstrates how to establish communication between pods inside a cluster.

**Review by dmuelle — CHANGES_REQUESTED** [review]
a few minor edit suggestions

**Comment by AustinSeto** [comment]
All suggested changes made, thank you @dmuelle

**Review by dmuelle — APPROVED** [review]
LGTM thanks!

**Pull request merged**